src/features/import/components/ImportDialog/Configure/Configuration/DateConfig.tsx

```diff
@@ -69,7 +69,11 @@ const DateConfig: FC<DateConfigProps> = ({ uiDataColumn }) => {
           </ListSubheader>
           {dateFormats.map((format) => (
             <MenuItem key={format} sx={{ paddingLeft: 4 }} value={format}>
-              {format}
+              <Msg
+                id={
+                  messageIds.configuration.configure.dates.dateFormats[format]
+                }
+              />
```

Comment:
Why localize this? The format is already hard-coded (because that's what we're allowing the user to pick here) so I reckon the examples could also be hard-coded.

Reply:
No reason other than my head was spinny I think 😵‍💫 I'll change it :)